data-prepper-pipeline-parser/src/main/java/org/opensearch/dataprepper/pipeline/parser/PipelinesDataflowModelParser.java

```diff
@@ -68,6 +69,17 @@ private PipelinesDataFlowModel parseStreamToPipelineDataFlowModel(final InputStr
         }
     }
 
+    private PipelinesDataFlowModel parseStreamToTemplateDataFlowModel(final InputStream configurationInputStream) {
```

Comment:
I think this change is no longer necessary. And actually, you can probably revert this entire file.

Reply:
addressed in the latest pr